src/Pact/Eval.hs

```diff
@@ -1228,7 +1231,7 @@ evalUserAppBody :: Def Ref -> ([Term Name], FunType (Term Name)) -> Info -> Gas
                 -> (Term Ref -> Eval e (Term Name)) -> Eval e (Term Name)
 evalUserAppBody _d@Def{..} (as',ft') ai g run = guardRecursion fname (Just _dModule) $
 #ifdef ADVICE
-  eAdvise ai (AdviceUser (_d,as')) $ dup $ appCall fa ai as' $ fmap (g,) $ run bod'
+  eAdvise ai (AdviceUser _d) $ dup $ appCall fa ai as' $ fmap (g,) $ run bod'
```

Comment:
Should we we s/_d/d since the name `d` is no longer unused? (Underscore idiom is for unused identifiers IIRC?)